pkg/graveler/errors.go

```diff
@@ -59,6 +59,8 @@ var (
 	ErrTooManyTries                 = errors.New("too many tries")
 	ErrSkipValueUpdate              = errors.New("skip value update")
 	ErrImport                       = wrapError(ErrUserVisible, "import error")
+	ErrReadOnlyRepository           = errors.New("read-only repository")
+	ErrWriteToReadOnlyRepository    = wrapError(ErrReadOnlyRepository, "cannot write to a read-only repository")
```

Comment:
Is there any case we should return `ErrReadOnlyRepository` as an error?
If not, can you please only one Err, having both errors when only one of them is intended to be used, confuses stuff